contributions/api/utils/mongoutils.py

```diff
@@ -30,8 +30,8 @@
 coll_contribution = db_contribution[cfg.CONTRIBUTION_COLL_NAME]  # set contribution collection
 # drop old indexes in contribution collection
 coll_contribution_indexes = coll_contribution.index_information().keys()
-# get all indexes except id and text index
-coll_contribution_old_indexes = [index for index in coll_contribution_indexes if index not in ['_id_', 'text_index']]
+# get old indexes
+coll_contribution_old_indexes = [index for index in coll_contribution_indexes if index in ['name', 'capabilities.name', 'talents.name']]
```

Comment:
Tested using Robo 3T that the name index is getting removed. Thanks.

In the future, as part of code cleanup, I think these lines of code could be converted to an init_db function called when the extractor starts.

Reply:
sounds good.